docs/_reporting/power-bi/README.md

```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
+| Invoice sections                                    |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| CSP customers (partner only)                        |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Management groups                                   |                ❌                 |                ❌                 |             ✅                     |              ❌               |
+| Subscriptions                                       |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Resource groups                                     |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Supports savings plans<sup>4</sup>                  |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Supports savings plan recommendations               |                ❌                 |                🔜                 |             ✅                     |              🔜               |
+| Supports multiple scopes                            |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Supports scopes in different tenants<sup>5</sup>    |          ⚠️<sup>5</sup>           |          ⚠️<sup>5</sup>           |             ❌                     |          ⚠️<sup>5</sup>(via Hubs) |
+| Faster data load times                              |                🔜                 |                ✅                 |             ✅                     |         ✅ (via Hubs)         |
+| Supports >$65M in cost details                      |                ❌                 |                ✅                 |             ❌                     |              ✅               |
+| Accessible outside of Power BI                      |                ✅                 |                ✅                 |             ✅                     |              ✅               |
+| Kusto Query Language (KQL) support                        |                ❌                 |                ❌                 |             ✅                     |              ❌               |
+| Native integration with cost optimization workbook |         ❌                 |                ❌                 |             ✅<sup>6</sup>                     |              ❌               |
+| Azure Data Explorer built-in dashboards                    |                ❌                 |                ❌                 |             ✅                     |              ❌               |
```

Comment:
This doesn't feel applicable in a Power BI doc.
```suggestion
```

Reply:
THat is why I suggested this doc to be somewhere else. This is a help me choose section for Hubs.

Reply:
But this isn't about hubs. Right now it's about the options available for using Power BI. If we want to expand it to include other reporting options, then those would need to be columns and not rows. I would remove this for now since we don't even have any dashboards available out of the box.